test/fixtures/cinema_halls.yml

```diff
@@ -0,0 +1,21 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  capacity: 1
+  rails: MyString
+  g: MyString
+  model: MyString
+  cinema_hall: MyString
+  name: MyString
+  capacity: 1
+
+two:
+  name: MyString
+  capacity: 1
+  rails: MyString
+  g: MyString
+  model: MyString
+  cinema_hall: MyString
+  name: MyString
+  capacity: 1
```

Comment:
Hm, it seems that something is duplicated here. E.g. `capacity` and `name`. And what is `g`?